test/e2e/serial/workload_placement_test.go

```diff
@@ -0,0 +1,186 @@
+	e2efixture "github.com/openshift-kni/numaresources-operator/test/utils/fixture"
+	e2enrt "github.com/openshift-kni/numaresources-operator/test/utils/noderesourcetopologies"
+	"github.com/openshift-kni/numaresources-operator/test/utils/nrosched"
+	"github.com/openshift-kni/numaresources-operator/test/utils/objects"
+	e2ewait "github.com/openshift-kni/numaresources-operator/test/utils/objects/wait"
+)
+
+var _ = Describe("[serial][disruptive][scheduler] workload placement", func() {
+	var fxt *e2efixture.Fixture
+	var nroSchedObj *nropv1alpha1.NUMAResourcesScheduler
+	var schedulerName string
+	var nrtList nrtv1alpha1.NodeResourceTopologyList
+	var nrts []nrtv1alpha1.NodeResourceTopology
+
+	BeforeEach(func() {
+		var err error
+		fxt, err = e2efixture.Setup("e2e-test-workload-placement")
+		Expect(err).ToNot(HaveOccurred(), "unable to setup test fixture")
+
+		nroSchedObj = &nropv1alpha1.NUMAResourcesScheduler{}
+		err = fxt.Client.Get(context.TODO(), client.ObjectKey{Name: nrosched.NROSchedObjectName}, nroSchedObj)
+		Expect(err).ToNot(HaveOccurred(), "cannot get %q in the cluster", nrosched.NROSchedObjectName)
+
+		schedulerName = nroSchedObj.Status.SchedulerName
+		Expect(schedulerName).ToNot(BeEmpty(), "cannot autodetect the TAS scheduler name from the cluster")
+
+		err = fxt.Client.List(context.TODO(), &nrtList)
+		Expect(err).ToNot(HaveOccurred())
+
+		tmPolicy := nrtv1alpha1.SingleNUMANodeContainerLevel
+		nrts = e2enrt.FilterTopologyManagerPolicy(nrtList.Items, tmPolicy)
+		if len(nrts) < 2 {
+			Skip(fmt.Sprintf("not enough nodes with policy %q - found %d", string(tmPolicy), len(nrts)))
+		}
+
+		// Note that this test, being part of "serial", expects NO OTHER POD being scheduled
+		// in between, so we consider this information current and valid when the It()s run.
+	})
+
+	AfterEach(func() {
+		err := e2efixture.Teardown(fxt)
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	Context("cluster with multiple worker nodes suitable", func() {
+		It("[test_id:47575] should make a pod with two gu containers land on a node with enough resources on a specific NUMA zone, each container on a different zone", func() {
+			hostsRequired := 2
+
+			// note we hardcode the values we need here and when we pad node.
+			// This is ugly, but automatically computing the values is not straightforward
+			// and will we want to start lean and mean.
+
+			pod := objects.NewTestPodPause(fxt.Namespace.Name, "testpod")
+			pod.Spec.SchedulerName = schedulerName
+			pod.Spec.Containers = append(pod.Spec.Containers, pod.Spec.Containers[0])
+			pod.Spec.Containers[0].Resources.Limits = corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("2"),
+				corev1.ResourceMemory: resource.MustParse("256Mi"),
+			}
+			pod.Spec.Containers[1].Resources.Limits = corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("3"),
+				corev1.ResourceMemory: resource.MustParse("384Mi"),
+			}
+			totalRes := e2enrt.ResourcesFromGuaranteedPod(*pod)
```

Comment:
shouldn't this pod to have the same amount of memory and cpu both in required and limits to be considered guaranteed?

Reply:
setting the limits only makes the platform set the request equal to limits, it's a common shortcut: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits